### ISSUE TITLE
RNC Crash

### ISSUE THREAD

**Issue by lfoliveir4** [react-native-clipboard/clipboard#88]
## Environment
react-native run-android

## Platforms
Android

## Versions

- Android: latest
- iOS: latest
- react-native-netinfo: no contain
- react-native: 0.63.3
- react: 16.13.1

## Description
When I do the build to emulate the android, with the following command: react native run-android the cliboard library returns the error attached

## Reproducible Demo
npx react-native run-android

![image](https://user-images.githubusercontent.com/32423942/107722011-cd5ce200-6cbc-11eb-95a3-35064b41c9cd.png)

**Comment by jigs611989** [comment]
Any update here?

**Comment by MTayyabMehar** [comment]
Same issue, please resolve it.

**Comment by mattThousand** [comment]
Same issue

**Comment by GNT-Gustavo** [comment]
I had a react native community clipboard module in my project trying to use the same name, causing the crash.
Can anyone share their package.json

**Comment by ShivamGoyal26** [comment]
Yes, I have got the same issue too, please help me... is anyone here with the solution?

**Comment by GNT-Gustavo** [comment]
My issue was having react-native-community/clipboard, trying to use ClipboardModule. Ended up using that package instead of this one, but removing either one should should solve the issue.

**Comment by ShivamGoyal26** [comment]
> My issue was having react-native-community/clipboard, trying to use ClipboardModule. Ended up using that package instead of this one, but removing either one should should solve the issue.

Yes it did, Thanks a lot I just remove the other one in the package.json and it solved the problem :)
